src/actions/admin.ts

```diff
@@ -22,129 +21,115 @@ export function AdminActions<ActionsBase extends Constructor<TinlakeParams>>(Bas
+      if (!this.contract('ASSESSOR')?.wards) return false
+      return (await this.contract('ASSESSOR').wards(user)).toBN().toNumber() === 1
     }
 
-    // lender permissions (note: allowance operator for default deployment)
     canSetMinimumJuniorRatio = async (user: string) => {
       if (!this.contract('ASSESSOR')?.wards) return false
       return (await this.contract('ASSESSOR').wards(user)).toBN().toNumber() === 1
     }
 
-    canSetInvestorAllowanceJunior = async (user: string) => {
-      if (!this.contract('JUNIOR_OPERATOR')?.wards) return false
-      return (await this.contract('JUNIOR_OPERATOR').wards(user)).toBN().toNumber() === 1
+    canAddToJuniorMemberList = async (user: string) => {
+      if (!this.contract('JUNIOR_MEMBERLIST')?.wards) return false
+      return (await this.contract('JUNIOR_MEMBERLIST').wards(user)).toBN().toNumber() === 1
     }
 
-    canSetInvestorAllowanceSenior = async (user: string) => {
-      if (!this.contract('SENIOR_OPERATOR')?.wards) return false
-      if (!(this.contractAddresses['SENIOR_OPERATOR'] !== ZERO_ADDRESS)) return false
-      return (await this.contract('SENIOR_OPERATOR').wards(user)).toBN().toNumber() === 1
+    canAddToSeniorMemberList = async (user: string) => {
+      if (!this.contract('SENIOR_MEMBERLIST')?.wards) return false
+      if (!(this.contractAddresses['SENIOR_MEMBERLIST'] !== ZERO_ADDRESS)) return false
+      return (await this.contract('SENIOR_MEMBERLIST').wards(user)).toBN().toNumber() === 1
     }
 
+    // REV: not used, but can be left
     canSetLoanPrice = async (user: string) => {
       if (!this.contract('COLLECTOR')?.wards) return false
       return (await this.contract('COLLECTOR').wards(user)).toBN().toNumber() === 1
     }
 
-    // ------------ admin functions borrower-site -------------
-    existsRateGroup = async (ratePerSecond: string) => {
-      const rateGroup = getRateGroup(ratePerSecond)
-      const actualRate = (await this.contract('PILE').rates(rateGroup)).toBN()
-      return !actualRate.isZero()
+    // ------------ admin functions lender-side -------------
+    setMinimumJuniorRatio = async (ratio: string) => {
+      // Source: https://github.com/ethereum/web3.js/issues/2256#issuecomment-462730550
+      return this.pending(
+        this.contract('ASSESSOR').file(web3.fromAscii('maxSeniorRatio').padEnd(66, '0'), ratio, this.overrides)
+      )
     }
 
-    initRate = async (ratePerSecond: string) => {
-      const rateGroup = getRateGroup(ratePerSecond)
+    setMaximumJuniorRatio = async (ratio: string) => {
       // Source: https://github.com/ethereum/web3.js/issues/2256#issuecomment-462730550
       return this.pending(
-        this.contract('PILE').file(web3.fromAscii('rate').padEnd(66, '0'), rateGroup, ratePerSecond, this.overrides)
+        this.contract('ASSESSOR').file(web3.fromAscii('minSeniorRatio').padEnd(66, '0'), ratio, this.overrides)
```

Comment:
This should inverse the ratio, right? Something like below, but I'm not 100% sure on the exact syntax.

```suggestion
        this.contract('ASSESSOR').file(web3.fromAscii('minSeniorRatio').padEnd(66, '0'), 1 * 10 ** 27 - ratio, this.overrides)
```